### PR TITLE
Documentation update to explain in more detail how to locally build.

### PR DESCRIPTION
**Pull request by AaronRobson**
Please feel free to squash the commits.

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -33,6 +33,8 @@
 # Note: this build does not sign the binaries
 # It's up to the consumer of the binaries to sign them
 
+# Calling with --skip-snk will skip applying strong names to Assembies.
```
Comment (kc284) [inline build.sh]: L missing in assemblies.

**Comment by MihaelaStoica** [comment]
While we are here, let's also update the [MAINTAINERS](https://github.com/xenserver/dotnet-packages/blob/master/MAINTAINERS)

File: build.sh (hunk 1 above)
Comment (AaronRobson) [reply build.sh]: Thank you, I have corrected it in a new commit.

**Comment by AaronRobson** [comment]
Changes to the MAINTAINERS file have been added.
Please see the following for the equivalent change to the xenadmin project:
https://github.com/xenserver/xenadmin/pull/2288

**Pull request merged**